### ISSUE TITLE
Test methods in Techniques are not aligned with the SC 1.4.4 Resize Text definition.

### ISSUE THREAD

**Issue by ArnaudDelafosse** [w3c/wcag#2923]
The SC 1.4.4 Resize Text says "[...] text can be resized [...] **up to** 200 percent [...]" and again in [the SC's Understanding](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html) "Content satisfies the Success Criterion if it can be scaled **up to** 200% [...]" which, to my understanding, means that the test should _theoretically_ be conducted in every incrementation (percentage) up to and including 200%.
In practice, most of us must probably test the browsers's own graphic zoom and/or text resizing incrementations (e.g. 110%, 120%, 133%, 150%, 170% and 200% in Firefox or 110%, 125%, 150%, 175% and 200% in Chrome).

However in [Technique G178](https://www.w3.org/WAI/WCAG21/Techniques/general/G178) and [Technique F94](https://www.w3.org/WAI/WCAG21/Techniques/failures/F94) (and it might also be elsewhere) it is said "Check that the text size can be increased to 200%" and "Check that the text [...] can be resized to at least 200%", possibly implying that testing **only** at 200% will be sufficient to conform to SC 1.4.4.

Is it just me or does that need to be clarified by using "...**up to** 200%" everytime it is needed?

**Comment by detlevhfischer** [comment]
Since the normative text does not specify that the resizing has to be linear (i.e., it does not rule out that at certain viewport widths the CSS **reduces** font size slightly, leading to a more moderate increase than you would see without CSS changes to font size) the only thing that remains to be tested is if there is **any** setting (usually via Browser zoom) where 200% text size without loss of content or functionality can be achieved. (It has been noted that this may be counterproductive for text that is already large at the default size, but no watertight wording to modify the normative text has come forward so far,)

**Comment by JAWS-test** [comment]
@detlevhfischer I am unsure if it is in accordance with the SC to require only 200% zoom, although the SC says otherwise. When the SC was written, as far as I know, there were no media queries that could respond to font size adjustment in the browser or OS with jumps in scaling, i.e. when SC 1.4.4 was written, it was assumed that resizing was linear. This has changed, but we would have to react to this by adapting SC or at least the understandings and techniques - and discuss beforehand in which form this adaptation should take place

**Comment by ArnaudDelafosse** [comment]
I'm not sure I totally understand you answer @detlevhfischer nor why MQ would come into the equation, so let me clarify my question and explain first what made me realise there may be a problem.

I often hear/read auditors talking about testing text resizing "**at** 200%" (only) for conformance to SC 1.4.4 Resize Text. 
But not only do the SC + Understanding say "**up to** 200%", there is no doubt (to me at least) that the _spirit_ of this SC (or the actual intent if you prefer) is to make sure that users that need to see a bigger text (between 100% and _up to_ 200%) can do so without AT and use either a browser setting, an OS setting or an functionality that's provided by the site author (A- A+ buttons etc.) 
So it would make sense that auditors do test, as much as possible, intermediate states between ..well, 100 % and 200 %. Regardless of MQs and viewport size considerations.

So my question is: shouldn't tests in Techniques use the same condition as the SC's normative text in order to: 
- be logical
- avoid confusion with certain auditors?

**Comment by detlevhfischer** [comment]
> the spirit of this SC (or the actual intent if you prefer) is to make sure that users that need to see a bigger text (between 100% and up to 200%) can do so without AT and use either a browser setting

@ArnaudDelafosse I am not contesting this. If there is no way to scale up text linearly (note this is not a frequent thing) it would still be not a failure of the normative requirement for the reasons given in my first response. Since the normative text does not specify what behavior would pass or fail the "up to" bit, it is not something that can be used for a FAIL just because you think it should fail. 

It is certainly good to give constructive comments in audit reports in cases where intermediate sizes are not (easily) available. As I said, I believe this is likely a pretty rare situation - possibly apart from size adjustments to larger text (headings) that will already be easier to read due to their defaut size.

**Comment by ArnaudDelafosse** [comment]
What I think is a fail is: with a text enlargement at 150% (in Firefox with the "Zoom text only" checked) AND a graphical (global) zoom at 150% in Chrome, some text or functionality that disappears in some way or another, even though it is perfectly fine at 200%.  
I admit this happens rarely but I have come across it a few times.
Wouldn't you fail that? Anyone else?

**Comment by JAWS-test** [comment]
> Wouldn't you fail that? 

Yes, of course

**Comment by scottaohara** [comment]
related to the topic of increments: https://github.com/w3c/wcag/issues/1857

>I admit this happens rarely but I have come across it a few times.
Wouldn't you fail that? Anyone else?

well... would it though if using browser zoom, instead of text zoom, didn't produce this gap?  see: https://github.com/w3c/wcag/issues/2169 and unmerged PR https://github.com/w3c/wai-easy-checks/issues/116

**Comment by detlevhfischer** [comment]
The similar discussions at #1857 and #2169 clarify that resizing of text needs to work using **any one method**. The absence of specific normative requirements for resizing increments means that the content passes 1.4.3 if text can be resized to 200% by any method.

@ArnaudDelafosse  I think [the issue you point out above](https://github.com/w3c/wcag/issues/2923#issuecomment-1386022762) occuring when using two text resizing methods simultaneously cannot constitute a failure as long as any single method does the trick.

Can we close this issue?

**Comment by ArnaudDelafosse** [comment]
Well I don't think that the issue should be closed yet. 
Some very interesting stuff in this conversation, no doubt, but my initial question (see issue title + my 1st comment) about having "...up to 200%" in some places and "...to 200%" elsewhere, with the consequence that some auditors only audit AT 200% and nowhere in between (regardless of method or incrementation) remains. This means that sometimes, the SC 1.4.4 is marked as Pass instead of Fail.
Don't you think this is worth addressing?

**Comment by ArnaudDelafosse** [comment]
> The similar discussions at #1857 and #2169 clarify that resizing of text needs to work using **any one method**. The absence of specific normative requirements for resizing increments means that the content passes 1.4.3 if text can be resized to 200% by any method.
> 
> @ArnaudDelafosse I think [the issue you point out above](https://github.com/w3c/wcag/issues/2923#issuecomment-1386022762) occuring when using two text resizing methods simultaneously cannot constitute a failure as long as any single method does the trick.

And I'm afraid this out of the scope of my initial question. Furthermore, we agree on that.

**Comment by ArnaudDelafosse** [comment]
While I'm here and over a year after... any chance that the word "up" (as in "...**up** to 200%") may be added wherever necessary (at least in [Technique G178](https://www.w3.org/WAI/WCAG21/Techniques/general/G178) and [Technique F94](https://www.w3.org/WAI/WCAG21/Techniques/failures/F94)) for clarification?

**Comment by GreggVan** [comment]
interesting.  We can't change the SC at this point we certainly could change the title and wording of the technique.   There is no question that that technique would satisfy the SC even though some might say it satisfies it and more.   There is no prohibition about a technique not going beyond as long as it satisfies.   (though it should have been "up to" in the SC, and that was the intent, and that is how it should be read as well)

**Comment by ArnaudDelafosse** [comment]
Thanks @GreggVan but I never asked for the SC to be changed :), only the _Techniques_ to use the same wording as the SC: that is "...**up** to 200%"

**Comment by GreggVan** [comment]
@ArnaudDelafosse    - in that case -- yep.  no brainer to do that.    (It was "at least in techniques x and y  that confused me.)
but UP TO is better -- and since that is the SC then it is even better

**Comment by ArnaudDelafosse** [comment]
That's great news, thanks. Hopefully this will clarify things for everyone.

**Comment by patrickhlauke** [comment]
Filed https://github.com/w3c/wcag/pull/3986
